src/odemis/acq/feature.py

```
@@ -471,6 +486,161 @@ def _create_fibsem_filename(filename: str, acq_type: str) -> str:
+
+        # Build privacy-preserving payload — generic keys, no names or filenames.
+        payload: dict = {
+            "status": feature.status.value,
+            "stage_position": dict(stage_pos),
+            "fm_focus_position": dict(feature.fm_focus_position.value),
+        }
+
+        def _get_raw(stream: "Stream") -> Optional["model.DataArray"]:
+            try:
+                return stream.raw[0] if stream.raw else None
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)